docs/tornjak-ui-api-documentation.md

```diff
@@ -49,7 +49,59 @@ Below are UI pages of Tornjak with their respective paths. For details on a spec
 
 
 # 3. Tornjak User Interface (UI) Interaction with API Endpoints
-## 3.1. Agent API’s
+
+## 3.1. Tornjak API’s
+### - [Healthcheck](https://pkg.go.dev/google.golang.org/grpc/health/grpc_health_v1#HealthCheckRequest)
```

Comment:
Why is this referencing health check from grpc instead of SPIRE?

Reply:
This is the health check tool that SPIRE uses - they import it directly in their implementation, so I think for us to access programmatically, this is what we have to do. Maybe I should put a note here?